app/templates/courses/teacher-classes-view.jade

```diff
@@ -127,3 +131,70 @@ mixin archivedClassRow(classroom)
     .col-xs-2
       .class-links.pull-right
         a.unarchive-classroom.text-h6(data-i18n='teacher.unarchive_class' data-classroom-id=classroom.id)
+
+mixin teacher-quests
+  - var numComplete = 0;
+  - var totalQuests = 0;
```

Comment:
It's faster to just enumerate the object once, and I'm not sure this adds any more clarity.